org.lflang/src/org/lflang/ast/ToLf.java

```diff
@@ -691,6 +710,55 @@ public String defaultCase(EObject object) {
             object.getClass().getName()
         ));
     }
+    
+    /**
+     * Wraps a multi-line String based on the current state of lineWrap and indentLvl.
+     * If lineWrap is 0, returns orgString. Otherwise, breaks lines such that if possible, each line has less than
+     * (lineWrap - (indentLvl * INDENTATION) % lineWrap) characters, only breaking at spaces.
+     * @param orgString The String to wrap.
+     * @return The wrapped String.
+     */
+    private String wrapLines(String orgString) {
+        if (lineWrap == 0) return orgString;
+        return orgString.lines().map(this::wrapIndividualLine).collect(Collectors.joining(System.lineSeparator()));
+    }
+    
+    /**
+     * Wraps a single line of String based on the current state of lineWrap and indentLvl. See wrapLines().
+     * @param line The line to wrap.
+     * @return The wrapped line.
+     */
+    private String wrapIndividualLine(String line) {
+        int wrapLength = lineWrap - (indentLvl * INDENTATION) % lineWrap;
```

Comment:
This use of `%` ensures that `wrapLength >= 0`, but it looks strange and I am not sure it implements a behavior that users would expect.